### ISSUE TITLE
it borked mah console :<

### ISSUE THREAD

**Issue by tostercx** [mjswensen/themer-cmd#1]
Using themer-gui v0.15.0, Github Universe preset on Win 10 1703, build 15063.726

![image](https://user-images.githubusercontent.com/4965387/34416680-3e5463f0-ebfd-11e7-9e39-75749ab8c2d3.png)

themer-dark.reg:

```
Windows Registry Editor Version 5.00

[HKEY_CURRENT_USER\Console]
"ColorTable00"=dword:002D2923
"ColorTable01"=dword:00A4A2A0
"ColorTable02"=dword:00686661
"ColorTable03"=dword:00C2C0BF
"ColorTable04"=dword:005570F3
"ColorTable05"=dword:00B87350
"ColorTable06"=dword:00868480
"ColorTable07"=dword:00DFDFDE
"ColorTable08"=dword:004B4742
"ColorTable09"=dword:00AD9912
"ColorTable10"=dword:0082BB6E
"ColorTable11"=dword:0099B309
"ColorTable12"=dword:007C4EEF
"ColorTable13"=dword:00AB67A0
"ColorTable14"=dword:003295F7
"ColorTable15"=dword:00FDFDFD
"ScreenColors"=dword:007C4EEF
"PopupColors"=dword:002D2923
```

~~Trying to figure out how to get it back to default at least. Hmm...~~ Got defaults from another PC :)

**Comment by mjswensen** [comment]
Yikes! I will look into it.

**Comment by mjswensen** [comment]
What happens if you then run `color 01`?

**Comment by tostercx** [comment]
Yep color 01 / 02 etc fixes it. Any way to save it as a default?

**Comment by tostercx** [comment]
Removing `ScreenColors` from the reg file seems to fix it. Not sure what the setting does.

**Comment by mjswensen** [comment]
Nice find. I had incorrectly assigned a color to that option, rather than what it actually expects. Fix forthcoming 🛠

**Comment by mjswensen** [comment]
@tostercx OK, I just published version `1.0.2` which should fix this. Update the package, regenerate your `.reg` file and see if it works. Feel free to reopen if it doesn't!

**Comment by tostercx** [comment]
Yep, works! :)
Thanks for the quick fix
